_posts/2020-05-20-gsoc-students.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Google Summer of Code Students 2020
+---
+
+We are happy to anounce that MDAnalysis is hosting three [GSoC][gsoc]
+students this year -- @hmacdope, @cbouy, and @yuxuanzhuang. This is
+the first year that MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] and we are grateful to Google for granting us
+three student slots so that we can have _three exciting GSoC
+projects_.
+
+## Hugo MacDermott-Opeskin: Trajectory New Generation: the trajectory format for the future of simulation
+
+<img
+src="https://pbs.twimg.com/profile_images/1140166815919665152/D-x2Tvae_400x400.jpg"
+title="Hugo MacDermott-Opeskin" alt="Hugo MacDermott-Opeskin"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Trajectory storage has always proved problematic for the molecular
+simulation community, as large volumes of data can be generated
+quickly. Traditional trajectory formats suffer from poor portability,
+large file sizes and limited ability to include metadata relevant to
+simulation. The Trajectory New Generation format developed by the
+[Gromacs][] team represents the first trajectory format with small
+file sizes, metadata storage, archive integrity verification and
+user/software signatures. In [this
```

Comment:
More of a picky thing than anything (do ignore if you disagree):
This sentence starting with `In this project` is rather long and has ~ 4 "and"s.
Probably the easiest thing I could think to fix it is just to ditch the last part "and is the primary goal of this project"? Or if outlining that this is the main aim is important, then maybe change the starting portion of the sentence to "The main objective of this project..."